WalletWasabi.Fluent/Models/Wallets/TransactionModel.cs

```diff
@@ -59,6 +59,11 @@ public partial class TransactionModel : ReactiveObject
 
 	private Money GetAmount()
 	{
+		if (IsCoinjoin)
+		{
+			return Amount;
+		}
+
 		return Amount < Money.Zero
 			? Amount + (Fee ?? Money.Zero)
 			: Amount;
```

Comment:
This change can be tricky.

Now, that even a CJ has (will have once the response from backend arrives) a proper `Fee` property, we can't simply do
`Amount + (Fee ?? Money.Zero)`, because it would look like this `0.00017232 + 0.01489441`.

This would make the `DisplayAmount` to be a positive number, which would mean we gained money in the CJ. Which is not gonna happen.

So, if we know it's a CJ, we simply return the `Amount`, which represents the outgoing amount from my wallet.

Reply:
I have a testnet coinjoin from yesterday that has a positive amount, never noticed such thing before.
the `Fees` is used as positive amount in the history
could it be because of this?

Reply:
> it remained that way after switching back to master

If it remained the same, then it's not because of this PR.

Reply:
was afraid so, sorry for polluting